libs/feature/editor/src/lib/+state/editor.reducer.ts

```diff
@@ -24,6 +24,7 @@ export interface EditorState {
   changedSinceSave: boolean
   editorConfig: EditorConfig
   currentPage: number
+  hasRecordChanged: any[]
```

Comment:
Could we type this? Maybe as an object `{user: string, date: Date}` as proposed in my other comment?